config/routes.rb

```diff
@@ -1,11 +1,12 @@
 Rails.application.routes.draw do
 
-  get 'charges/create'
-
   resources :wikis
   resources :charges, only: [:new, :create]
 
   devise_for :users
+  resources :users, only: [:update, :show] do
+    post 'downgrade'
```

Comment:
@bjblock - I also added this within the routes as I was getting a no routes error when clicking on the downgrade button. Is this correct?